src/app/features/ledger/hooks/use-ledger-navigate.ts

```diff
@@ -141,15 +141,13 @@ export function useLedgerNavigate() {
       },
 
       cancelLedgerAction() {
-        // for send ordinal '..' brings you back to the `choose-fee` step but you lose context of the ordinal
-        const backLocation = location.pathname.match(RouteUrls.SendOrdinalInscription)
-          ? RouteUrls.Home
-          : '..';
+        // Use baseUrl to determine where to go on close
+        const baseUrl = `/${location.pathname.split('/')[1]}`;
 
-        return navigate(backLocation, {
+        return navigate(baseUrl, {
```

Comment:
@kyranjamie @edgarkhanzadian : As part of my work here I dug into trying to get specific actions to also cancel on the `Ledger` device but had no luck. Did either of you try that before? 

I see that we do close transport successfully when we have a successful transaction but I couldn't get it to work when the user has a cancelled transaction for example. 

During testing this, often I:
- initiated an action from Leather to Ledger
- cancelled the action on Leather
- observed my Ledger as locked but when I un-locked it was still awaiting my approval / rejection of the previous action

That meant that I was blocked from doing any other action until I unlocked my ledger and declined the previous action. 

I tried to use the `generic-ledger-utils` - `quitAppOnDevice` and also wrote my own similar code to try and exit out of these transactions but I couldn't get it working. 

It was failing with:
```
Failed to execute 'close' on 'USBDevice': An operation that changes interface state is in progress.
```

If I am interacting with Gamma for example, and start a JWT auth then close the Leather popup, the request stays open on my Ledger. 

I'm just curious if we have tried fixing this before as it would be a nice addition to exit the appropriate APP on Ledger if we are no longer performing an action.

Reply:
> trying to get specific actions to also cancel on the Ledger device but had no luck

Afaik, to cancel an operation the best you can do is just close the USB connection with `transport.close()`. If an action is pending on the device, I don't think there's anything you can do (unless closing the connection manually cancels it? though not sure)

Reply:
If you try and close the connection it errors with `Failed to execute 'close' on 'USBDevice': ..`

Thanks though, I was just looking to get some more knowledge on it